docs/protocol-design/work.md

```diff
@@ -14,16 +14,51 @@ Existing content:
 
 ---
 
-## Spam resistance focus
+## Spam resistance
+
+!!! quote ""
+	Proof-of-Work (PoW) is used as a method to deter spam transactions on the network. It is **not** used in the consensus mechanism.
+
+A spam transaction is a block broadcasted with the intention of saturating the network, reducing its available throughput for other network participants, or increasing the size of the ledger. Participants can compute the required PoW in the order of seconds. The cost of spamming the network increases linearly with the number of spam transactions, thus reducing the impact of spam transactions from theoretically infinite to a manageable amount.
 
 ---
 
-## PoW algorithm details
+## PoW in state blocks
+
+Every [state block](./blocks) includes a work field that must be correctly populated. A valid proof is obtained by randomly guessing a nonce such that:
+
+$$
+H(\text{nonce} || \text{x}) \ge \text{threshold}
+$$
+
+where $H$ is an algorithm, usually in the form of a hash function, $||$ is the concatenation operator, $threshold$ is a parameter of the network that relates to the resources spent to obtain a valid work, and $x$ is either:
+
+- The account's public key, in the case of the first block
+- The previous block's hash, otherwise
+
+The following image illustrates the process by which a valid proof is obtained.
+
+![generate-work](/images/whitepaper/generate-work.png)
```

Comment:
The diagram is a little confusing - this is the process for getting PoW for block 3 so perhaps that should be stated above. And it might also clear things up if the arrows pointed out of block 3 instead of block 2. Or you could have the arrows doing work for block 2 (so mention pervious as block 1 hash) and then have additional arrow coming out of block 3 with a dotted line mentioning the below process is repeated for block 3 but with block 2's hash.